aesara/link/jax/dispatch/basic.py

```diff
@@ -0,0 +1,284 @@
+import warnings
+from functools import singledispatch
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+from aesara.compile.ops import DeepCopyOp, ViewOp
+from aesara.configdefaults import config
+from aesara.graph import Constant
+from aesara.graph.fg import FunctionGraph
+from aesara.ifelse import IfElse
+from aesara.link.utils import fgraph_to_python
+from aesara.raise_op import CheckAndRaise
+from aesara.tensor.shape import Reshape, Shape, Shape_i, SpecifyShape, Unbroadcast
+from aesara.tensor.subtensor import (
+    AdvancedIncSubtensor,
+    AdvancedIncSubtensor1,
+    AdvancedSubtensor,
+    AdvancedSubtensor1,
+    IncSubtensor,
+    Subtensor,
+    indices_from_subtensor,
+)
```

Comment:
Do you want to make `dispatch` modules for these in a follow-up PR?  Regardless, we should create an issue for these.

Reply:
Do you mean `jax/dispatch/subtensor.py` and `jax/dispatch/shape.py`? I'm happy to do it in this PR.

Reply:
> Do you mean `jax/dispatch/subtensor.py` and `jax/dispatch/shape.py`? I'm happy to do it in this PR.

Yes, those two.

Reply:
I just pushed the changes.